tests/integration/it_roles_test.py

```diff
@@ -13,7 +13,7 @@ def test_create(client, db_session):
     response = client.post(url_for("role.create"), data={
         "name": "John"
     })
-    assert response.location.endswith(url_for('role.list_roles'))
+    assert response.location.endswith("/roles/")
```

Comment:
@marceloamadeu can you leave the old ``url_for`` call? Maybe the urls will change in the future, and then this test will fail since it would be hardcoded.

Reply:
@amihaiemil True if we use `url_for` will avoid future problems. Adjusted.